### ISSUE TITLE
AttributeError: 'PVOutput' object has no attribute 'addoutput'

### ISSUE THREAD

**Issue by Kugeleis** [yaleman/pvoutput#172]
Running the example async_addoutput.py fails with `AttributeError: 'PVOutput' object has no attribute 'addoutput'`

**Comment by yaleman** [comment]
Hi there, how are you invoking it, and from what path? The module definitely has that function...

**Comment by Kugeleis** [comment]
e.g. running sync_check_rate_limit.py from within the examples folder with my credentials in a pvoutput.json works well. This proves the setup works (python 3.10).

**Comment by yaleman** [comment]
You'd need to have the `pvoutput` module installed for that to run, can you please show the output of `python -m pip freeze | grep -i pvoutput`

**Comment by Kugeleis** [comment]
of course it's installed: `pvoutput==0.0.8
`

**Comment by yaleman** [comment]
I can't assume anything at this point due to the lack of context that's been provided, so not "of course". 😄 

I was interested in what version, because I really don't know how the code can possibly be throwing the error you're reporting. The example literally just runs [one of the functions in the core library](https://github.com/yaleman/pvoutput/blob/main/pvoutput/asyncio/__init__.py#L134) and that example and the function it calls haven't meaningfully changed since it was added to the repository.

If it gets to the line where it's trying to run `addoutput` then the code has successfully instantiated the class, so it's not that either. Can you please try one of the other async examples? 

Just to humour me, try updating your copy of the repo before hand. I assume you're using a clone of the main branch of `yaleman/pvoutput`... I haven't changed the functionality of the tests, just cleaned up typing and config loading (note the added `donation_made` field).

**Comment by Kugeleis** [comment]
OK, I pulled a fresh copy, set up my credentials and just tried `python sync_check_rat_limit.py` which runs just fine, the async version wants a donation :-)
For ` sync_addoutput.py ` I still get the error `AttributeError: 'PVOutput' object has no attribute 'addoutput'`.

**Comment by Kugeleis** [comment]
![pvoutput png](https://user-images.githubusercontent.com/9948777/164036273-adf74990-83de-4ceb-91e7-26762c3c796d.jpg)
VS Code even gives a context help for the function. Weired!

**Comment by yaleman** [comment]
Yeah, the fun with this whole thing is that there's clearly something going on in your environment that's messing with the outcome. Maybe try running it in a debugger to pull out state before it gets to the line that throws an error and go from there.

**Comment by yaleman** [comment]
v0.0.9 is released and it works now, apologies again 😞
